fairscale/nn/data_parallel/fsdp_optim_utils.py

```diff
@@ -156,7 +158,7 @@ def build_unflat_state_dict(
     unflat_state, global_to_local_id = _unflatten_optim_state(state, instance_list, world_pad_info, singleton_state)
     # Since there are no tensors in param_groups, deepcopy is fine
     param_groups = copy.deepcopy(param_groups)
-    num_params = sum([len(m._param_numels) for m in instance_list])  # type: ignore
+    num_params = sum([cast(int, m.num_params_managed) for m in instance_list])
```

Comment:
(nit) I'd comment that `cast` doesn't do anything besides please the `mypy` gods.
I had to scroll up to see where it was imported from.

Reply:
Good idea. done.